.github/workflows/publish-to-test-pypi.yaml

```diff
@@ -22,7 +22,7 @@ jobs:
           pip3 install .
           pip3 install .[pypi]
           pip3 install build
-          pip3 install setuptools>=64
+          pip3 install setuptools --upgrade
```

Comment:
Do you want to specify a min version as well? I guess not as this only ever has to work on a specifically configured runner.